.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
```

Comment:
in the other repos we are already using
rev: v4.4.0

Reply:
Done!